datarepo-clienttests/src/main/java/runner/DeploymentScript.java

```diff
@@ -0,0 +1,43 @@
+package runner;
+
+import java.util.List;
+import runner.config.ApplicationSpecification;
+import runner.config.ServerSpecification;
+
+public class DeploymentScript {
```

Comment:
Why not make this abstract?

Reply:
Good idea. I'll make both the DeploymentScript and TestScript classes abstract and include it in the PR I've currently got open.